src/AppBundle/Controller/ClientController.php

```diff
@@ -90,4 +100,42 @@ public function addAction(Request $request)
 
         return ['form' => $form->createView()];
     }
+
+    /**
+     * @Route("/client/verify", name="client_verify")
+     * @Template()
+     */
+    public function verifyAction(Request $request)
+    {
+        $user = $this->getUserWithData();
+
+        if (! $user->getIsCoDeputy()) {
+            return $this->redirect($this->generateUrl('client_add'));
+        }
+
+        $client = $this->getFirstClient();
+        $formClient = clone $client;
+        $formClient->setLastName('');
+        $formClient->setCaseNumber('');
+
+        $form = $this->createForm(new FormDir\ClientVerifyType(), $formClient);
+
+        $form->handleRequest($request);
+        if ($form->isValid()) {
+            if ((strToLower($client->getLastname()) === strToLower($form['lastname']->getData()))
+                && $client->getCaseNumber() === $form['caseNumber']->getData())
+            {
+                $user->setCoDeputyClientConfirmed(true);
+                $this->getRestClient()->put('user/'.$user->getId(), $user);
+                return $this->redirect($this->generateUrl('homepage'));
+            } else {
+                $form->get('lastname')->addError(new FormError('We could not match the client last name'));
+                $form->get('caseNumber')->addError(new FormError('We could not match the client case number'));
+            }
+        }
+
+        return [ 'form' => $form->createView()
```

Comment:
once merged we can run the `php-cs-fixed` with the config inside the repo to fix code syntax